### PR TITLE
improve github downloader on migrations

### PR DESCRIPTION
**Pull request by lunny**
The new interface will make code easier to understand.

### CODE REVIEW

File: modules/migrations/git.go [hunk 1]
```diff
@@ -53,17 +53,17 @@ func (g *PlainGitDownloader) GetReleases() ([]*base.Release, error) {
 	return nil, ErrNotSupported
 }
 
-// GetIssues returns issues according start and limit
-func (g *PlainGitDownloader) GetIssues(start, limit int) ([]*base.Issue, error) {
-	return nil, ErrNotSupported
+// GetIssues returns issues according page and PerPage
+func (g *PlainGitDownloader) GetIssues(page, PerPage int) ([]*base.Issue, bool, error) {
```
Comment (lafriks) [inline modules/migrations/git.go]: `pageSize` would be better name imho
```suggestion
func (g *PlainGitDownloader) GetIssues(page, pageSize int) ([]*base.Issue, bool, error) {
```

Comment (lunny) [reply modules/migrations/git.go]: The name is from github's API document. :)

Comment (lafriks) [reply modules/migrations/git.go]: Method argument then should be at least lowercase first letter

**Comment by lunny** [comment]
@lafriks done.

**Review by lafriks — APPROVED** [review]

**Review by adelowo — APPROVED** [review]

**Pull request merged**